addons/mail/tests/test_mail_channel.py

```diff
@@ -197,6 +201,27 @@ def test_channel_members(self):
         self.assertEqual(channel.message_partner_ids, self.env['res.partner'])
         self.assertEqual(channel.channel_partner_ids, self.env['res.partner'])
 
+    @users('employee')
+    @mute_logger('odoo.addons.mail.models.mail_mail', 'odoo.models.unlink')
+    def test_channel_chat_message_post_should_update_last_activity_time(self):
+        channel = self.env['mail.channel'].browse(self.test_channel.ids)
+        channel.write({'channel_type': 'chat'})
+        channel._action_add_members(self.partner_employee | self.partner_admin)
+        now = fields.Datetime.now()
+        with patch.object(fields.Datetime, 'now', lambda: now):
+            with self.mock_mail_gateway():
```

Comment:
Is that needed? We shouldn't need to care about emails in this test.

Reply:
it is needed because the with this context, notification and mail records will be cleared automatically. Otherwise other tests may fail.